api/health_check.go

```diff
@@ -15,10 +22,48 @@ type networkInfo struct {
 }
 
 type healthCheckResponse struct {
-	Network networkInfo `json:"network"`
+	CoreIndexer *coreIndexerHealth `json:"core_indexer"`
+	Network     networkInfo        `json:"network"`
+}
+
+type coreIndexerHealth struct {
+	LastIndexedBlock int64 `json:"last_indexed_block"`
+	BlockDiff        int64 `json:"block_diff"`
+}
+
+func (app *ApiServer) getCoreIndexerHealth(ctx context.Context) (*coreIndexerHealth, error) {
+	nodeInfo, err := app.auds.Core.GetNodeInfo(ctx, connect.NewRequest(&corev1.GetNodeInfoRequest{}))
+	if err != nil {
+		return nil, fiber.NewError(fiber.StatusInternalServerError, "Failed to get core node info")
+	}
+	chainHeight := nodeInfo.Msg.CurrentHeight
+
+	var indexerLastBlockHeight int64
```

Comment:
so the idea is that we use https://discoveryprovider.audius.co/health_check for the old indexer and https://api.audius.co/health_check for the new one?